GitUpKit/Views/GIAdvancedCommitViewController.m

```diff
@@ -50,6 +54,40 @@ @implementation GIAdvancedCommitViewController {
   BOOL _disableFeedback;
 }
 
+#pragma mark - Search
+- (void)setupSearch {
+  self.searchTextField.delegate = self;
+  self.searchTextField.placeholderString = NSLocalizedString(@"Filter files", nil);
```

Comment:
nit: don't have localization

Reply:
It’s fine to go that direction anyways, perhaps even encouraged. Makes any future translation easier.